h/accounts/schemas.py

```diff
@@ -203,13 +203,15 @@ class SocialLoginSignupSchema(colander.Schema):
     comms_opt_in = comms_opt_in_node()
 
 
-class EmailChangeSchema(CSRFSchema):
```

Comment:
Note that I'm removing the `CSRFSchema` base class, using Pyramid's `require_csrf=True` in the view configs instead.